src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { pipe, toUpper, trim } from 'ramda';
+import PropTypes from 'prop-types';
+
+import { Button } from '../Button';
+import { ButtonColors, ButtonVariants } from '../Button/Button.enums';
+import { Sizes, UserAvatarProps } from './UserAvatar.types';
+import {
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  getRadii,
+  pxToRem,
+} from '../../utils/helpers';
+import { UserAvatarSizes } from './UserAvatar.enums';
+
+const UserAvatarSmall = css`
+  padding: ${pxToRem(4, 2.5)};
+`;
+const UserAvatarMedium = css`
+  padding: ${pxToRem(8, 6.5)};
+`;
+
+const userAvatarSizes = {
+  [UserAvatarSizes.sm]: UserAvatarSmall,
+  [UserAvatarSizes.md]: UserAvatarMedium,
+};
+
+const StyledButton = styled(Button)<{ size: Sizes }>`
```

Comment:
I'm thinking if we should support a non-button variant (as `div` element, without hover, focus, active states)

Reply:
Hmm well the avatar component on MUI is non-interactive by default.  

Can we use an "interactive=false" attribute? By default true? Not sure if there is a use case in which would be used.

Reply:
We can leave it for now, but I expect that we will get a request to show this avatar e.g. in setting in the user profile. But we can easily add this later